### PR TITLE
Add St_John's_College_MS_34

### PR DESCRIPTION
**Pull request by stjohnscollegeoxford**
Made changes to encoding as discussed; added <bibl> elements throughout where it seemed appropriate, unsure if these should be kept in.

### CODE REVIEW

File: St_John's_College_MS_34.xml [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                     <origDate calendar="Gregorian" notBefore="1175" notAfter="1200">s. xii ex.</origDate>
+                     <origPlace>
+                        <country key="place_1000070">French</country>
+                     </origPlace>
+                  </origin>
+                  <provenance>
+                     <p>Two lines of Latin medical notes, perhaps verse (on a bit of vellum 85 mm x 135–50 mm, mounted on the front pastedown,<date notBefore="1275" notAfter="1300">s. xiii ex.</date>)</p>
+                  </provenance>
+                  <provenance>
+                     <q type="written">‘Liber Collegij Sanctj Iohannis Baptistae Oxon’ ex dono Magistrj<persName role="dnr" key="person_2833">Iohannis Lambe</persName> Registrarij Petriburgensis’</q>
```
Comment (holfordm) [inline St_John's_College_MS_34.xml:122]: A space is needed after "Magistrj"

File: St_John's_College_MS_34.xml [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+               </physDesc>
+               <history>
+                  <origin>
+                     <origDate calendar="Gregorian" notBefore="1175" notAfter="1200">s. xii ex.</origDate>
+                     <origPlace>
+                        <country key="place_1000070">French</country>
+                     </origPlace>
+                  </origin>
+                  <provenance>
+                     <p>Two lines of Latin medical notes, perhaps verse (on a bit of vellum 85 mm x 135–50 mm, mounted on the front pastedown,<date notBefore="1275" notAfter="1300">s. xiii ex.</date>)</p>
```
Comment (holfordm) [inline St_John's_College_MS_34.xml:119]: space needed after the comma and before the date element

File: St_John's_College_MS_34.xml [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+                     <layoutDesc>
+                        <layout>
+                           <dimensions type="column" unit="mm">
+                              <height quantity="205">205</height>
+                              <width min="53" max="55">53-55</width>
+                              </dimensions>
+                              <dimensions type="intercolumn" unit="mm">
+                                 <width quantity="11">11</width>
+                              </dimensions>
+                        </layout>
```
Comment (holfordm) [inline St_John's_College_MS_34.xml:89]: Could you add some text to explain the dimensions? e.g. "Column dimensions ....; .... between columns." As it stands, only the numbers will appear when the xml is converted to html for web display. (This only applies to dimensions in the layout section.)

**Comment by holfordm** [comment]
I've just suggested a couple of minor changes, otherwise looks excellent. The additional `date` elements aren't necessary but don't do any harm.

**Comment by andrew-morrison** [comment]
The file also needs to be moved to a subfolder within the _collections_ folder. And it would make things easier if the apostrophe were omitted from names of folders and files because, like spaces, it can cause problems when batch processing.

**Comment by holfordm** [comment]
Thanks @andrew-morrison  - @stjohnscollegeoxford within your "collections" folder, please create a new folder called "St_Johns_College", and then move this file into that folder, and rename the file without the apostrophe - it will look in the GitHub client as though you have deleted one file and created a new one, but that's ok.

**Comment by stjohnscollegeoxford** [comment]
Thank you both, I'll work on that this afternoon.

**Comment by holfordm** [comment]
The website is usually re-indexed at the end of the day on Thursday, and this record will be visible then. 
One other thing to consider is whether you would like the collection for St John's to read something like "St John's College (in progress)", just in case anyone should think the college only has one medieval manuscript!

**Comment by stjohnscollegeoxford** [comment]
That's perfect, thank you! Labelling the St John's Collection as 'in progress' would be great. Thank you for all your help with this so far.

**Pull request merged**